master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: IP Pool Manipulation
+---
+This document describes how to make common modifications to Calico's
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.  The
+`-o yaml` will output in a format that can be captured, manipulated, and then
+used to apply the desired changes.
+
+```shell
+$ calicoctl get ipPool -o yaml
+- apiVersion: v1
+  kind: ipPool
+  metadata:
+    cidr: 192.168.0.0/16
+  spec: {}
+```
+
+## Enabling IPIP and NAT Outgoing on a pool
+
+### 1. Retrieve current IP Pool config
+
+```shell
+$ calicoctl get ipPool -o yaml > pool.yaml
+```
+
+### 2. Modify IP Pool config
+
+Edit the saved `pool.yaml` as desired.  See 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+for the options that can be updated, added, or removed.
+
+Here we'll modify the pool's spec to enable IP-IP and nat-outgoing.
```

Comment:
The wording here "Here we'll modify" makes it sound like "For example" ... whereas the purpose of this section is to enable IPIP and NAT outgoing.